Data/DataModel/Prerequisites.cs

```diff
@@ -12,11 +12,21 @@ public class Prerequisites
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int ID { get; set; }
 
+        /// <summary>
+        /// The puzzle ID that depends on others.
+        /// </summary>
+        public int PuzzleID { get; set; }
+
         /// <summary>
         /// The puzzle that depends on others.
         /// </summary>
         public virtual Puzzle Puzzle { get; set; }
 
+        /// <summary>
+        /// A potential prerequisite ID of the puzzle named in Puzzle, which may need to be solved before Puzzle will be unlocked.
+        /// </summary>
```

Comment:
Is there a reason you need a variable for both the ID and the Puzzle when the ID can be referenced from the Puzzle?